contexts/credentials/v2

```diff
@@ -0,0 +1,214 @@
+        "verificationMethod": {"@id": "https://w3id.org/security#verificationMethod", "@type": "@id"}
+      }
+    },
+
+    "EcdsaSecp256r1Signature2019": {
+      "@id": "https://w3id.org/security#EcdsaSecp256r1Signature2019",
+      "@context": {
+        "@version": 1.1,
+        "@protected": true,
+
+        "id": "@id",
+        "type": "@type",
+
+        "challenge": "https://w3id.org/security#challenge",
+        "created": {"@id": "http://purl.org/dc/terms/created", "@type": "http://www.w3.org/2001/XMLSchema#dateTime"},
+        "domain": "https://w3id.org/security#domain",
+        "expires": {"@id": "https://w3id.org/security#expiration", "@type": "http://www.w3.org/2001/XMLSchema#dateTime"},
+        "jws": "https://w3id.org/security#jws",
+        "nonce": "https://w3id.org/security#nonce",
+        "proofPurpose": {
+          "@id": "https://w3id.org/security#proofPurpose",
+          "@type": "@vocab",
+          "@context": {
+            "@version": 1.1,
+            "@protected": true,
+
+            "id": "@id",
+            "type": "@type",
+
+            "assertionMethod": {"@id": "https://w3id.org/security#assertionMethod", "@type": "@id", "@container": "@set"},
+            "authentication": {"@id": "https://w3id.org/security#authenticationMethod", "@type": "@id", "@container": "@set"}
+          }
+        },
+        "proofValue": "https://w3id.org/security#proofValue",
+        "verificationMethod": {"@id": "https://w3id.org/security#verificationMethod", "@type": "@id"}
+      }
+    },
+
+    "Ed25519Signature2018": {
+      "@id": "https://w3id.org/security#Ed25519Signature2018",
+      "@context": {
+        "@version": 1.1,
+        "@protected": true,
+
+        "id": "@id",
+        "type": "@type",
+
+        "challenge": "https://w3id.org/security#challenge",
+        "created": {"@id": "http://purl.org/dc/terms/created", "@type": "http://www.w3.org/2001/XMLSchema#dateTime"},
+        "domain": "https://w3id.org/security#domain",
+        "expires": {"@id": "https://w3id.org/security#expiration", "@type": "http://www.w3.org/2001/XMLSchema#dateTime"},
+        "jws": "https://w3id.org/security#jws",
+        "nonce": "https://w3id.org/security#nonce",
+        "proofPurpose": {
+          "@id": "https://w3id.org/security#proofPurpose",
+          "@type": "@vocab",
+          "@context": {
+            "@version": 1.1,
+            "@protected": true,
+
+            "id": "@id",
+            "type": "@type",
+
+            "assertionMethod": {"@id": "https://w3id.org/security#assertionMethod", "@type": "@id", "@container": "@set"},
+            "authentication": {"@id": "https://w3id.org/security#authenticationMethod", "@type": "@id", "@container": "@set"}
+          }
+        },
+        "proofValue": "https://w3id.org/security#proofValue",
+        "verificationMethod": {"@id": "https://w3id.org/security#verificationMethod", "@type": "@id"}
+      }
+    },
+
+    "RsaSignature2018": {
```

Comment:
After looking into this more in the perspective of the v3-unstable security vocab, I'd prefer to drop these out of the V2 context here. Instead opting to have VCs use the security context as a second context or to list it as a secondary context in this so that only this context needs to be included still.

Reply:
@kdenhartog I believe that the VC WG objected to the idea of suggesting multiple / open world contexts... ironic, given the purpose of linked data....

To me this is similar to DID Core issue, we should have something like:

```
"@context": [
"https://example.com/stable/credential/formats/v1",
"https://extensions.example.com/unstable/credential/formats/v1",
]
```

Reply:
In that case we'd want to define this context such that it imports in the new v3 security context. Technically we're doing the same thing, but the data model of the VCs wouldn't show it.

Reply:
+1 to separating those terms out into the v3 security context instead of duplicating them here.

Reply:
Since the purpose of this PR is just to add two new properties to the core data model, should we either

1) Just simplify this PR to address the spec changes and not update (create a new context)
2) Create a new context as a copy of v1 with the new properties added (what this PR is currently doing)
3) Create a new context that tackles other changes we want to see in v2 (e.g refactoring how we handle security related terms)